src/pages/Encounters/EncounterShow.tsx

```
@@ -196,6 +196,7 @@
                     data-cy={`tab-${tab}`}
                     className={tabButtonClasses(props.tab === tab)}
                     href={`${tab}`}
+                    data-cy={`encounter-${tab.toLowerCase()}-tab`}
```

ESLint

[error] No duplicate props allowed

(react/jsx-no-duplicate-props)